Remove owner from materialized views metadata table in documentation (#21062)
trinodb/trino: docs/src/main/sphinx/connector/system.md

  - Freshness of data in the storage table. Queries on the materialized view
    access the storage table if not `STALE`, otherwise the `definition` is used
    to access the underlying data in the source tables.

> * - `owner`

Reviewer: `last_fresh_time` column is missing.
https://github.com/trinodb/trino/blob/0edd4b429f4f6d3fbba83bbe0545a46dfa03d6a4/core/trino-main/src/main/java/io/trino/connector/system/MaterializedViewSystemTable.java#L60-L72

Author: Thanks for taking a look, @ebyhr. I have added the `last_fresh_time` column and a description.

  - Freshness of data in the storage table. Queries on the materialized view
    access the storage table if not `STALE`, otherwise the `definition` is used
    to access the underlying data in the source tables.
* - `last_fresh_time`

>   - Indicates the last time the materialized view was refreshed.

Reviewer 2: ```suggestion
  - Date and time of the last refresh of the materialized view..
```
